.github/workflows/lock.yml

```diff
@@ -0,0 +1,15 @@
+name: 'lock closed issues/PRs'
+on:
+  schedule:
+    - cron: '*/2 * * * *'
+jobs:
+  lock:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: dessant/lock-threads@v2
```

Comment:
Let's fork this, `v2` here can be a tag or branch, both which are mutable, so even if the action is legit now, they can always change it from under us.

I plan on using GH settings to enforce valid sources of actions in the future (I'm unsure if these can be applied org wide or not).

Reply:
Forked to https://github.com/getsentry/lock-threads. Public or private?

Reply:
I'm going to test the fork on a smaller repo and then I'll return here.

Reply:
public is fine

Reply:
No need for a fork, using a SHA should be enough to mitigate this as they are immutable

Reply:
I'm against forking every third-party thing we use if there are alternatives to that approach.

Reply:
Pinned via SHA in fb3e5cdac1d38459a598cdc3bc1799870ac6b4d5. You good with that, @billyvg?